core/src/main/java/org/apache/carbondata/core/cache/dictionary/ReverseDictionaryCache.java

```diff
@@ -168,6 +168,10 @@ public ReverseDictionaryCache(CarbonLRUCache carbonLRUCache) {
             CacheType.REVERSE_DICTIONARY));
   }
 
+  @Override public void put(DictionaryColumnUniqueIdentifier key, Dictionary value) {
+
+  }
+
```

Comment:
override the method in abstract class and in the implementation throw UnsupportedException for both reverse and forward cache